### PR TITLE
Switch over CI on GKE to another project.

### PR DESCRIPTION
**Pull request by spxtr**
Until 1.3.7 is released, at least. It might be worth keeping it there permanently, in which case I will need to document/automate the new firewall rule better.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/555)

### CODE REVIEW

File: ciongke/Makefile [hunk 1]
```diff
@@ -36,16 +39,18 @@ JENKINS_SECRET_FILE = ${HOME}/jenkins
...
 create-cluster:
 	gcloud -q container --project "$(PROJECT)" clusters create ciongke --zone "$(ZONE)" --machine-type "$(MACHINE_TYPE)" --scope "https://www.googleapis.com/auth/compute","https://www.googleapis.com/auth/devstorage.full_control","https://www.googleapis.com/auth/logging.write","https://www.googleapis.com/auth/servicecontrol","https://www.googleapis.com/auth/service.management" --num-nodes "$(NUM_NODES)" --network "default" --enable-cloud-logging --enable-cloud-monitoring
-	kubectl create secret generic hmac-token --from-file=hmac=$(HOOK_SECRET_FILE)
-	kubectl create secret generic oauth-token --from-file=oauth=$(OAUTH_SECRET_FILE)
-	kubectl create secret generic jenkins-token --from-file=jenkins=$(JENKINS_SECRET_FILE)
-	make test-pr-image --no-print-directory
-	make hook-image --no-print-directory
-	make hook-deployment --no-print-directory
-	make hook-service --no-print-directory
+	@kubectl create secret generic hmac-token --from-file=hmac=$(HOOK_SECRET_FILE)
```
Comment (rmmh) [inline ciongke/Makefile]: does this automatically use the appropriate credentials from the container creation right before?

Comment (spxtr) [inline ciongke/Makefile]: I don't understand the question. This is right when you create the cluster, there won't be any containers yet.

Comment (rmmh) [inline ciongke/Makefile]: Err, sorry, from the `container cluster` creation, which specifies `--project`.

Comment (spxtr) [inline ciongke/Makefile]: Yes.

**Pull request merged**